src/lib/mod.rs

```diff
@@ -64,6 +64,15 @@ pub fn is_gzip_path<P: AsRef<Path>>(p: &P) -> bool {
     is_path_with_extension(p, GZIP_EXTENSIONS)
 }
 
+/// The set of file extensions to treat as ZSTD compressed
+/// The second extension is redundant yes, only for is_path_with_extension()
+const ZSTD_EXTENSIONS: [&str; 2] = ["zst", "zst"];
+
+/// Returns true if the path ends with a recognized ZSTD file extension
+pub fn is_zstd_path<P: AsRef<Path>>(p: &P) -> bool {
+    is_path_with_extension(p, ZSTD_EXTENSIONS)
+}
+
```

Comment:
should we remove these methods and constants, and then use those in fgoxide?

Reply:
The only place I see `is_fastq_path`, `is_gzip_path`, and `is_zstd_path` is the `spawn_reader` function. Could be better to move/merge all constants and methods related to these into fgoxide and then call them from the fgoxide `Io` module

Reply:
Done